mmv1/third_party/terraform/tests/resource_container_cluster_test.go.erb

```diff
@@ -3920,6 +3920,9 @@ resource "google_container_cluster" "primary" {
     kalm_config {
 	  enabled = false
 	}
+	gcs_fuse_csi_driver_config {
```

Comment:
nit: can we fix the spacing on this (and the line above it) please, to be lined up with kalm_config